rhombus/scribblings/ref-array.scrbl

```diff
@@ -192,10 +196,61 @@ mutable and immutable arrays, while @rhombus(MutableArray, ~annot) and
 }
 
 
+@doc(
+  fun Array.get(arr :: Array, n :: NonnegInt) :: Any
+){
+
+ Returns the @rhombus(n)th element of @rhombus(arr) (starting from
+ @rhombus(0)), like the implicit @rhombus(#%index) form (usually
+ written with @brackets).
```

Comment:
This is precise, but I think it would help to show `arr[n]` at the start of the explanation (in addition to the examples), since someone who jumps to this documentation probably won't know about the weird name `#%index` and how it it written with `[]`, but they'll probably instantly recognize `arr[n]`.